.pylintrc

```diff
@@ -10,13 +10,15 @@
 #    C0330: Wrong hanging indentation before block (add 4 spaces)
 #    C0326: Bad whitespace
 # 5. fixme (left 'TODO' lines)
-# 6. The following require installing the python modules imported in the source code
+# 6. logging-fstring-interpolation (forbids f-strings in logging functions)
+# 7. missing-function-docstring (Missing function or method docstring)
+# 8. The following require installing the python modules imported in the source code.
+#    Add these if you don't want to include all dependencies into the dev environment:
 #    import-error ("Unable to import")
 #    no-member
 #    c-extension-no-member
-# 7. missing-function-docstring (Missing function or method docstring)
 
-disable=f-string-without-interpolation,inherit-non-class,too-few-public-methods,C0330,C0326,fixme,import-error,no-member,c-extension-no-member,missing-function-docstring
```

Comment:
Did you mean to remove the missing-function-docstring one?

Reply:
Yep, I don't know why I disabled this in the first place. I think we can add some docstrings into every python file. Except for __init__.py, which I hopefully pylint will understand :)